plugins/ray.yaml

```diff
@@ -3,37 +3,37 @@ kind: Plugin
 metadata:
   name: ray
 spec:
-  version: v1.2.2
+  version: v1.3.0-rc.1
```

Comment:
@MortalHappiness does this remove v1.2.2 from the index?

Reply:
I believe that `krew-index` only stores the latest version, and the other YAML files also contain only one version.

Reply:
ah I wasn't aware of that, in that case I think we should only update krew index on official releases, not release candidates, what do you think?

Reply:
+1. I found that the reviewer has already closed this PR.